docs/readmes/lte/deploy_install_docker.md

```diff
@@ -74,10 +74,16 @@ bash agw_install_docker.sh
 
 Once you see the output `Reboot this machine to apply kernel settings`, reboot your AGW host.
 
+After reboot become root user again:
+
+```bash
+sudo -i
+```
+
 Create `control_proxy.yml` file with your orc8r details:
 
 ```bash
-cat << EOF | sudo tee /var/opt/magma/configs/control_proxy.yml
+cat << EOF | tee /var/opt/magma/configs/control_proxy.yml
```

Comment:
I wonder why this change would be necessary?
In the end that means that also the commands further below, i.e. the `docker compose up` and `exec` would be run with elevated privileges. I would doubt that this is necessary and would rather avoid having docker run as root.

Reply:
We are already becoming `root` user on the command above, so we don't need sudo commands.

Reply:
instead , you could have added `sudo` in front of `docker-compose` and `docker` calls

Reply:
> I wonder why this change would be necessary?

some of the configuration for docker compose is owned by rood and has 400 permissions

Reply:
Then we will have to run sudo with each command. That will be too much sudo. If we want that then I can update the docs.

Reply:
> some of the configuration for docker compose is owned by rood and has 400 permissions

Would it than not be better to allow read access to those configuration in order to _not_ run docker as root user?
I would believe getting read access to yet another config file is much less of a security concern than execution docker as root user.

Reply:
I have updated docs with sudo commands

Reply:
The actual question has not been answered. Executing docker as root has security implications and should not be done without a proper justification.

Reply:
@maxhbr what should we do here?

Reply:
>  I would believe getting read access to yet another config file is much less of a security concern than execution docker as root user.

being able to run docker is equivalent to being root. And `sudo` in front of `docker-compose` or `docker` does not elevate there permissions further.